Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -50,19 +50,33 @@
 require "missing_formula"
 require "digest"
 
+require_relative "../cli_parser"
+
 module Homebrew
   module_function
 
   def audit
     inject_dump_stats!(FormulaAuditor, /^audit_/) if ARGV.switch? "D"
     Homebrew.auditing = true
+    args, _unnamed_args = HomebrewCli.options do
+      option :strict, "strict", type: :boolean, default: false
```

Comment:
The second argument should also be named and just default to a modified version of the symbol name,